src/main/java/uk/ac/ebi/eva/utils/FileUtils.java

```diff
@@ -1,5 +1,7 @@
 package uk.ac.ebi.eva.utils;
```

Comment:
Cof cof this file has changes from Mut cof cof... and he didn't put the license cof cof...